projects/packages/my-jetpack/_inc/data/products/use-install-standalone-plugin.ts

```diff
@@ -3,8 +3,22 @@ import { REST_API_SITE_PRODUCTS_ENDPOINT } from '../constants';
 import { QUERY_INSTALL_PRODUCT_KEY } from '../constants';
 import useSimpleMutation from '../use-simple-mutation';
 import useProduct from './use-product';
+import type { APIFetchOptionsWithQueryParams } from '../use-simple-mutation';
+import type { UseMutateFunction } from '@tanstack/react-query';
 
-const useInstallStandalonePlugin = ( productId: string ) => {
+export type InstallAction = UseMutateFunction<
+	void,
+	Error,
+	APIFetchOptionsWithQueryParams,
+	unknown
+>;
+
+type InstallStandalonePlugin = ( productId: string ) => {
```

Comment:
We could still call it `UseInstallStandalonePluginFunction`. Doesn't matter so much that is longer imo, but it's more descriptive. Similar to `UseMutateFunction` we're importing above in this PR.